langchain/chains/question_answering/load.ts

```diff
@@ -11,13 +11,23 @@ import {
   DEFAULT_COMBINE_QA_PROMPT,
 } from "./map_reduce_prompts";
 
-interface qaChainParams {
-  prompt?: PromptTemplate;
-  combineMapPrompt?: PromptTemplate;
-  combinePrompt?: PromptTemplate;
-  type?: string;
-}
-export const loadQAChain = (llm: BaseLLM, params: qaChainParams = {}) => {
+type chainTypeName = "stuff" | "map_reduce";
```

Comment:
I would give this type a more strict name and follow upper casing for it too (given that its a type):

`QuestionAnsweringChainType` for example